app/controllers/api/users.php

```diff
@@ -1023,41 +1023,6 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $response->dynamic($user, Response::MODEL_USER);
     });
 
-App::patch('/v1/users/:userId/verification')
-    ->desc('Update Email Verification')
-    ->groups(['api', 'users'])
-    ->label('event', 'users.[userId].update.verification')
-    ->label('scope', 'users.write')
-    ->label('audits.event', 'verification.update')
-    ->label('audits.resource', 'user/{request.userId}')
-    ->label('audits.userId', '{request.userId}')
-    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
```

Comment:
don't we need this endpoint anymore ? I don't see any other endpoint where we are able to update email verification

Reply:
It was in two places probably some merge issues.